restlight-jaxrs-provider/src/main/java/io/esastack/restlight/jaxrs/adapter/MessageBodyWriterAdapter.java

```diff
@@ -43,11 +43,11 @@ public MessageBodyWriterAdapter(Providers providers) {
 
     @SuppressWarnings("unchecked")
     @Override
-    public HandledValue<Void> writeTo(ResponseEntity entity,
+    public Result<Void, Void> writeTo(ResponseEntity entity,
                                       ResponseEntityChannel channel,
                                       RequestContext context) throws Exception {
         if (entity.response().entity() == null) {
-            return HandledValue.failed();
+            return Result.err();
```

Comment:
Err is a more general expression. In fact, if we really want to use a more accurate expression, failed is not very appropriate too. We should express it with unsupported. But for reuse, we reuse result.err(), instead of starting a new stove.in this case, there is no need to be too tangled about whether the expression is completely accurate, and the meaning is roughly the same.